packages/mimikatz.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,49 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'mimikatz'
+$category = 'Password Attacks'
+
+
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+$name = "Mimikatz"
+$path = Join-Path ${Env:TOOL_LIST_DIR} 'Password Attacks'
+
+# Remove files from previous zips for upgrade
+VM-Remove-PreviousZipPackage ${Env:chocolateyPackageFolder}
+
+$toolsDir = Join-Path "${Env:RAW_TOOLS_DIR}" $name
+if (-Not (Test-Path $toolsDir)) {
+  New-Item -Path $toolsDir -ItemType Directory -Force | Out-Null
+}
+
+$url = "https://github.com/gentilkiwi/mimikatz/releases/download/2.2.0-20220919/mimikatz_trunk.7z"
```

Comment:
You'll likely want to use `VM-Install-From-Zip` (see: https://github.com/mandiant/VM-Packages/blob/main/packages/common.vm/tools/vm.common/vm.common.psm1#L269).

Here's an example using it: https://github.com/mandiant/VM-Packages/blob/main/packages/gobuster.vm/tools/chocolateyinstall.ps1

In general, I recommend reading over all the helper functions in https://github.com/mandiant/VM-Packages/blob/main/packages/common.vm/tools/vm.common/vm.common.psm to know what's available

Reply:
I looked over the help tool `VM-Install-From-Zip`, but found that its current options are unable to reach the nest files for Mimikatz zip file structure (there is only an option to go down one nested folder of the same name as the tool). 

I may look into modifying the helper tool for future installations, but it looks like Mimikatz is an outlier.